src/miniscript/mod.rs

```diff
@@ -64,37 +64,40 @@ pub struct Miniscript<Pk: MiniscriptKey, Ctx: ScriptContext> {
 }
 
 /// `PartialOrd` of `Miniscript` must depend only on node and not the type information.
-/// The type information and extra_properties can be deterministically determined
-/// by the ast.
+///
+/// The type information and extra_properties can be deterministically determined by the AST.
```

Comment:
In 33c04e24d8e9bbf7265d5d36e3b1ec53f2c26762:

While we're touching this, let's replace "can be deterministically determined" with "are implied" :P.